lib/browser/init.js

```diff
@@ -103,14 +104,15 @@ require('./guest-window-manager')
 // Now we try to load app's package.json.
 let packagePath = null
 let packageJson = null
-const searchPaths = ['app', 'app.asar', 'default_app.asar']
-for (packagePath of searchPaths) {
+for (packagePath of asar.getSearchPath()) {
```

Comment:
I hope it is the only place where we do search since it is very critical (see comment for method `GetSearchPath` in the `atom_api_asar.cc`).